src/one/nio/http/QueryParameterIterator.java

```diff
@@ -0,0 +1,127 @@
+package one.nio.http;
```

Comment:
New source files should include standard license header

Reply:
Fixed.